datasets/README.md

```diff
@@ -108,3 +105,66 @@ python3 spd_datasets.py \
     --hub_repository=diarizers-community/callhome \
 ```
 
+
+## 2. Generate a synthetic dataset compatible with diarizers: 
```

Comment:
Would be great to explain here why you'd want to use synthetic data (over real data) and how it scales up to an arbitrary amount of training data.